src/style.css

```diff
@@ -0,0 +1,20 @@
+div {
+  height: 100px;
+  width: auto;
+}
+
+.red {
```

Comment:
The name of the element class should make a sense. It should describe what the role of the element is on this page or describe its state. In the case with the name _red_, _green_, maybe in the future, the color of your block will be changed, so you have to rename all classes on the page.